streams/src/test/java/org/apache/kafka/test/MockProcessor.java

```diff
@@ -41,16 +44,16 @@ public MockProcessor() {
         delegate = new MockApiProcessor<>();
     }
 
-    @SuppressWarnings("unchecked")
     @Override
-    public void init(final ProcessorContext context) {
-        super.init(context);
-        delegate.init((org.apache.kafka.streams.processor.api.ProcessorContext<Object, Object>) context);
+    public void init(ProcessorContext<KOut, VOut> context) {
+        Processor.super.init(context);
```

Comment:
Why adding `Processor.` ? Seems unnecessary.

Reply:
I think this was IDEA trying to be smart and me looking over it.